### PR TITLE
Kitchen Spike Hotfix

### LINKED ISSUE #11971: After being meat spiked you are permanently upside down.
So yo, after you get meatspiked you get turned upside down permanently.  Stunning and such doesn't seem to reset your sprite.

image

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
To prevent forever upside down mobs (and extra "the"s in messages)
fixes #11971

### CODE REVIEW

**Comment by Razharas** [comment]
How urgend is this?

**Comment by KorPhaeron** [comment]
@Razharas 

Depends on how much you care about people walking around upside down. It's purely a visual thing, but it looks really dumb.

**Comment by KorPhaeron** [comment]
Admins also can't fix it via var stuff so far as I know

**Comment by Razharas** [comment]
Yes our varedit doesnt provide the meanings to change matrixes since they are special byond snowflake

**Comment by xxalpha** [comment]
>removing features

Come on.

**Comment by Lijah99** [comment]
"off of" isn't proper grammar :^(

**Comment by PKPenguin321** [comment]
fixes #11971

**Comment by RemieRichards** [comment]
@KorPhaeron @Razharas 
Modify a human's ```lying_prev``` to an angle and the human will rotate that many degrees.
it's technically not a feature or fix, but it does work.

File: code/game/objects/structures/kitchen_spike.dm [hunk 1]
```diff
@@ -110,12 +110,12 @@
 		if(!M.buckled)
 			return
 		var/mob/living/L = buckled_mob
-		var/matrix/m120 = matrix(L.transform)
+		var/matrix/m120 = matrix()
 		m120.Turn(360)
```
Comment (RemieRichards) [inline code/game/objects/structures/kitchen_spike.dm]: I think this is the issue actually.
turn 360 is going back to where you started, isn't it?

**Comment by Miauw62** [comment]
another bug of the year candidate

**Comment by Razharas** [comment]
@KorPhaeron cmon do what rr said

**Comment by KorPhaeron** [comment]
@Razharas 

Alright

**Pull request merged**